.github/workflows/linux.yml

```diff
@@ -0,0 +1,25 @@
+name: Linux desktop CI
+
+on:
+  pull_request: 
+    types: [synchronize, review_requested, ready_for_review]
+
+concurrency:
+  group: ${{ github.head_ref }}
+  cancel-in-progress: true
+
+jobs:
+  test-building:
+    if: github.event.pull_request.draft == false
+    runs-on: [ self-hosted, desktop-kdrive-tests ]
+
+    steps:
+      - name: Checkout the code
```

Comment:
Which code? (Maybe it is clear in the context of this job, but I only see this narrow excerpt while reviewing).

Reply:
This checks out the repository on the pull request HEAD. I renamed to `Checkout the PR`, feel free to suggest another name